icu4j/main/classes/core/src/com/ibm/icu/util/ULocale.java

```diff
@@ -877,8 +877,9 @@ public static String[] getISOCountries() {
     }
 
     /**
-     * Returns a list of all 2-letter language codes defined in ISO 639.
-     * Can be used to create Locales.
+     * Returns a list of all 2-letter language codes defined in ISO 639, plus additional
```

Comment:
Sorry for the delay, I didn't see it.

I think I would stay away from "determined to be useful" and just mention the facts (what is returned) and point to BCP 47.

What do you think about:
```
Returns a list of all unique language codes defined in ISO 639. They can be 2 or 3 letter codes, as defined by [BCP 47, section 2.2.1](https://www.ietf.org/rfc/bcp/bcp47.html#section-2.2.1).
```

Reply:
Done